Sources/SwifterSwift/Foundation/DateExtensions.swift

```diff
@@ -487,6 +487,20 @@ public extension Date {
         return timeIntervalSince1970
     }
 
+    /// SwifterSwift: Current day in a year
+    var dayInYear: Int {
+        return calendar.ordinality(of: .day, in: .year, for: self)!
```

Comment:
This could be nil by any chance?

Reply:
I don't thinks so. The documentation states that :
Returns
The ordinal number of smaller within larger at the time specified by date. Returns nil if larger is not logically bigger than smaller in the calendar, or the given combination of components does not make sense (or is a computation which is undefined).
Day is always smaller than year, so nil is not thrown. 
What do you think ? Did I understand correctly ?

Reply:
I don't have deep knowledge about calendars, so my question is more about that depending on the calendar used if this could by any chance crash. I assume our tests use always gregorian, so for this, I think it is never nil. But honestly no idea about others 😅

Reply:
In my branch for upgrading to Xcode 11.1 I found that this crashed with `.nanoseconds` -> `.seconds` but only on Linux builds. You never know when some implementation can go horribly wrong.
That being said, I think it's safe to assume that you can get which number of day in the year a specific date is, regardless of the calendar.